src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java

```diff
@@ -0,0 +1,33 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import co.aikar.commands.CommandHelp;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.HelpCommand;
+import co.aikar.commands.annotation.Subcommand;
+import co.aikar.commands.annotation.Syntax;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class UsageCommand extends MultiverseCoreCommand {
+    public UsageCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+    }
+
+    @HelpCommand
+    @Subcommand("help")
+    @CommandPermission("multiverse.core.help")
+    @CommandCompletion("@commands:mv")
```

Comment:
Maybe add  `@range|@commands:mv` here to show that you can specify a page

Reply:
from experience, typing a single number on the keyboard is easier than tabbing through a range of numbers